modules/benchmark/R/load_data.R

```diff
@@ -34,13 +34,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    key   <- get_key("https://bd-api.ncsa.illinois.edu",username,password)
+    token <- get_token("https://bd-api.ncsa.illinois.edu",key)
+    #output_path = where are we putting converted file?
+    converted.data.path <- convert_file(url = "https://bd-api.ncsa.illinois.edu", input_filename = data.path, 
+                                        output = "csv", output_path, token)
+    if (is.na(converted.data.path)){
+      PEcAn.utils::logger.error("Converted file was not returned from Brown Dog")
+    }
+    #not doing anything about mimetypes not convertible by BD right now
+    fcn <- match.fun(fcn2)
+    data.path <- converted.data.path
   } else {
-    PEcAn.utils::logger.warn(paste0("no load_data for current mimetype ",mimetype," - converting using browndog"))
+    PEcAn.utils::logger.warn(paste0("Brown Dog is currently unable to perform conversion from ",mimetype," to a PEcAn usable format")
```

Comment:
missing parenthesis at the end. Also as a FYI, the logger functions do not require a paste0, but do this implicitly. So you can simply do a : `logger.debug("hello", "world")` and it will print "hello world".